site/content/docs/manifest/backend-service.md

```diff
@@ -121,6 +121,29 @@ Grace period within which to provide containers time to bootstrap before failed
 
 <div class="separator"></div>
 
+<a id="entrypoint" href="#entrypoint" class="field">`entrypoint`</a> <span class="type">String or Array of Strings</span>  
+Override the default entrypoint in the image. 
+```yaml
+# String version.
+entrypoint: "/bin/entrypoint --p1 --p2"
+# Alteratively, as an array of strings.
+entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
+
+<div class="separator"></div>
+
+<a id="command" href="#command" class="field">`command`</a> <span class="type">String or Array of Strings</span>  
+Override the default command in the image.
+
```

Comment:
nit: I think we should have the same comments as above. It's redundant, but this is sort of a menu, so people may only read the entries they're looking into.

Reply:
Yess fixed now! I committed efe's suggestions directly from here and forgot to fix the rest ughhh